openapi.yml

```
@@ -743,6 +769,106 @@ components:
           type: string
           example: 2008/0545166T
           description: An identifier from the Police National Computer (PNC)
+    Sentence:
+      type: object
+      properties:
+        dataSource:
+          type: string
+          enum: [NOMIS, NDELIUS]
+          example: NOMIS
+          description: Which upstream API the sentence originates from
```

```suggestion
          description: Which legacy system the sentence originates from
```

Thinking "upstream API" would be "Prison API" and "legacy system" is "NOMIS"?